docs/components/Avatar.js

```diff
@@ -2,24 +2,29 @@ import Image from "next/image";
 
 export const Avatar = ({ name, picture, twitterUsername }) => {
   return (
-    <div className="flex items-center justify-center">
-      <Image
-        src={picture}
-        height={32}
-        width={32}
-        layout="fixed"
-        priority={true}
-        className="w-full rounded-full"
-        alt={name}
-      />
+    <div className="flex items-center md:justify-start flex-shrink-0  ">
```

Comment:
Nit, extra white space at the end of this string (feels like something prettier should be catching?)

Not blocking, just interesting - I'm going to check our prettier setup now.